src/components/DeleteModal.js

```diff
@@ -1,60 +1,47 @@
-import Button from './Button.js';
 import styled from 'styled-components';
 
+import { ModalButton } from './Button.js';
+
 export default function DeleteModal(props) {
   if (!props.show) {
```

Comment:
here you could destructure you show in `DeleteModal({show})` and then `if (!show) {`

Reply:
well you are right, especially as i already handed it over haha🤩